Cards/Effects/EffectDamage.gd

```diff
@@ -0,0 +1,12 @@
+class_name EffectDamage extends EffectBase
+
+# @Override
+func apply_effect(caster: Entity, target: Entity, value: int) -> void:
+	var target_damage_data: DealDamageData = DealDamageData.new()
+	target_damage_data.damage = value
+	target_damage_data.caster = caster
+	target.get_health_component().deal_damage(target_damage_data)
+
+# TODO : what to do for Status that inflige damage over time ?
+	# Should they call an EffectDamage when they trigger each turn ?
+	# or should they directly call deal_damage on the target ?
```

Comment:
The thing about EffectDamage it adds another layer of interaction. It's useful for making cards, but when the call is to deal damage
The status already have a countdown in themselves to know when it triggers. And generally speaking, a status does one thing (it doesn't trigger multiple things, so it doesn't really make sense for it to have effects, because it's not a card)

Reply:
Understood! Think it was just my misunderstanding of not knowing Effect vs. Status. If you want to resolve this thread feel free to do so, unless you wanted other people to chime in.

Reply:
Yeah there is some confusion right now because not everyone knows the different classes (which is why I opened #32 because it will be needed)

Reply:
I realized Status already inflict damage by calling deal_damage, so no need to change anything actually